### PR TITLE
chore: increase unit test coverage for CIDocument filterUtils

### PR DESCRIPTION
**Pull request by H-Fin**
#### What do these changes do/fix?
Adds on to the unit test for the CIDocument utils file `filterUtils.ts` for higher test coverage of our code.

https://zenhub.ibm.com/app/workspaces/discovery-unified-5728bf39d2c20f776e05d0ef/issues/watson-discovery/docviz-squad-issue-tracker/234

#### How do you test/verify these changes?
`yarn test watch:filterUtils` and see that tests pass.

Run `yarn test:coverage` and view the coverage of the CIDocument/utils/filterUtils.ts file to see the results are statements: 91.67%, branches: 90.91%, and functions: 100%.

### CODE REVIEW

File: packages/discovery-react-components/src/components/CIDocument/utils/__tests__/filterUtils.spec.ts [hunk 1]
```diff
@@ -56,14 +63,17 @@ describe('filterUtils', () => {
       filter = updateFilter(filter, filterPartA2);
       filter = updateFilter(filter, filterPartB1);
       filter = updateFilter(filter, filterPartB2);
+      filter = updateFilter(filter, filterPartC1);
 
       expect(filter.FILTER_GROUP_A.length).toBe(2);
       expect(filter.FILTER_GROUP_B.length).toBe(1);
-      expect(filter.FILTER_GROUP_C).toBe(undefined);
+      expect(filter.FILTER_GROUP_C.length).toBe(0);
+      expect(filter.FILTER_GROUP_D).toBe(undefined);
```
Comment (anibapat) [inline packages/discovery-react-components/src/components/CIDocument/utils/__tests__/filterUtils.spec.ts]: You can directly check for `.toBeUndefined()` here and in other place down, which is recommended over referring to `undefined` directly.

**Comment by H-Fin** [comment]
> Looks good. I asked Heather about getting this file up to full 100% coverage, because it's one more branch / statement. If it ends up being too much work, this PR can be merged in as is.

Newest commit covers that statement. Percentages are now statements: 100%, branches: 90.91%, and functions: 100%.

**Comment by H-Fin** [comment]
> Looks good. I asked Heather about getting this file up to full 100% coverage, because it's one more branch / statement. If it ends up being too much work, this PR can be merged in as is.

Newest commit covers that statement. Everything is now at 100%.

**Comment by ColbyJohnIBM** [comment]
I reviewed this again with the updates. Still looks good, and now at 100% coverage. :100:

**Pull request merged**